### PR TITLE
default style adjustment

### PR DESCRIPTION
**Pull request by BSd3v**
adjusted default style to be applied to the div as hardcoded instead of in the default props

### CODE REVIEW

**Comment by alexcjohnson** [comment]
Ah interesting - yes, this makes sense. Let's just add a test to ensure that both `height` and `width` can be overridden, separately or together, via the `style` prop. Unless we already have one that I'm not seeing?

**Comment by AnnMarieW** [comment]
Yes - based on this issue:  https://community.plotly.com/t/ag-grid-with-multi-page-apps/74728/15?u=annmariew

File: src/lib/fragments/AgGrid.react.js [hunk 1]
```diff
@@ -1164,7 +1164,11 @@ export default class DashAgGrid extends Component {
         }
 
         return (
-            <div id={id} className={className} style={style}>
+            <div
+                id={id}
+                className={className}
+                style={{...{height: '400px', width: '100%'}, ...style}}
```
Comment (alexcjohnson) [inline src/lib/fragments/AgGrid.react.js:1170]:
```suggestion
                style={{height: '400px', width: '100%', ...style}}
```

**Review by alexcjohnson — APPROVED** [review]
💃

**Comment by mathman79** [comment]
Does this work correctly with ```dashGridOptions={"domLayout": "autoHeight"}```? I've been explicitly omitting `"height"` in the `style` because of this, but since moving from 2.0.0rc2 to 2.0.0 my tables look weird.

**Comment by BSd3v** [comment]
This should be working.

Do you have an MRE to test?

**Comment by AnnMarieW** [comment]
@mathman79 You can find an example of it working with 2.0.0 in the docs here:  https://dashaggrid.pythonanywhere.com/layout/grid-size
and as @BSd3v  mentioned a minimal example that shows the issue would help.

**Comment by mathman79** [comment]
Ah, thanks a lot. I have to add ```{"height": "100%"}``` now to override the default. Thanks for the quick response.

**Pull request merged**